coinlib_flutter/windows/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+# The Flutter tooling requires that developers have a version of Visual Studio
+# installed that includes CMake 3.14 or later. You should not increase this
+# version, as doing so will cause the plugin to fail to compile for some
+# customers of the plugin.
+cmake_minimum_required(VERSION 3.14)
+
+# Project-level configuration.
+set(PROJECT_NAME "coinlib_flutter")
+project(${PROJECT_NAME} LANGUAGES CXX)
+
+# Invoke the build for native code shared with the other target platforms.
+# This can be changed to accommodate different builds.
+# add_subdirectory("${CMAKE_CURRENT_SOURCE_DIR}/../src" "${CMAKE_CURRENT_BINARY_DIR}/shared")
+# TODO re-enable the above line if/when the CMake process is made Windows-compatible.  See also below.
+
+# List of absolute paths to libraries that should be bundled with the plugin.
+# This list could contain prebuilt libraries, or libraries created by an
+# external build triggered from this build file.
+set(coinlib_flutter_bundled_libraries
+  # Defined in ../src/CMakeLists.txt.
+  # This can be changed to accommodate different builds.
+  # $<TARGET_FILE:coinlib_flutter>
+  # TODO re-enable the above line if/when the CMake process is made Windows-compatible.  See also above.
+  ${CMAKE_BINARY_DIR}/../../secp256k1.dll
```

Comment:
Is this correct with respect to the build_windows_crosscompile script producing a `libsecp256k1.dll` vs a `secp256k1.dll`?  Renaming file to test with yours.

Reply:
I take it that the name needs to be changed to `secp256k1.dll` when copied.

I'll be away for the rest of today, I'll be back tomorrow.